weaviate/collections/aggregations/aggregate.py

```diff
@@ -25,18 +25,21 @@
     _MetricsDate,
     _MetricsNumber,
     _MetricsInteger,
-    # _MetricsReference, # Aggregate references currently bugged on Weaviate's side
+    _MetricsReference,
```

Comment:
Unless something changed, they are still bugged. It is not the protocol that is bugged, but the implementation

Reply:
Yeh, it has changed; Marcin identified the issue and fixed it. I'll re-add a test here to verify